eblob_kit.py

```diff
@@ -1060,92 +1155,114 @@ def check(self, verify_csum, fast):
         return self._valid
 
     @staticmethod
-    def recover_index(data, destination, overwrite=False):
+    def recover_index(data, destination, overwrite=False, move_back=False):
```

Comment:
Why isn't it check the resulting blob?

Reply:
Index file only should be checked as blob not touched here. ```BlobRepairer``` has ```check_index``` method, but there no any way to construct it only for index.

May be modify ```BlobRepairer``` to make it possible to run ```check_index``` without any blob access?

Reply:
Implemented post-check for indexes in ```recover_index```.